src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -10853,8 +10853,30 @@ bool Unit::InitTamedPet(Pet* pet, uint8 level, uint32 spell_id)
             creature->SetLootRecipient(nullptr);
     }
 
-    if (isRewardAllowed && creature && creature->GetLootRecipient())
-        player = creature->GetLootRecipient();
+    if (isRewardAllowed && creature)
```

Comment:
are these changes supposed to be in this pr ?

Reply:
git history seems broken (this change is actually merged into TC). will have to reset